spec/models/card_spec.rb

```diff
@@ -1,188 +1,338 @@
-require 'rails_helper'
+require "rails_helper"
 
 describe Card do
-  it 'create card with empty original text' do
-    card = Card.create(original_text: '', translated_text: 'house', user_id: 1,
-                       block_id: 1)
-    expect(card.errors[:original_text]).to include('Необходимо заполнить поле.')
-  end
+  let(:card) { Card.new(original_text: "дом", translated_text: "house",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.